.zshrc

```diff
@@ -0,0 +1 @@
+export PATH=$PATH:/Users/celinechoi/.yarn/bin
```

Comment:
ah sry i think this was because of me, but could you go ahead and delete this whole file